akka-http2-support/src/test/scala/akka/http/impl/engine/http2/Http2ServerSpec.scala

```diff
@@ -432,29 +432,38 @@ class Http2ServerSpec extends AkkaSpecWithMaterializer("""
         sendFrame(DataFrame(TheStreamId, endStream = false, ByteString("0" * 512001))) // more than default `incoming-stream-level-buffer-size = 512kB`
         expectRST_STREAM(TheStreamId, ErrorCode.FLOW_CONTROL_ERROR)
       }
-      "not leak stream if request entity is not fully pulled when connection dies" inAssertAllStagesStopped new WaitingForRequestData {
+      "fail stream if request entity is not fully pulled when connection dies" inAssertAllStagesStopped new WaitingForRequestData {
         sendDATA(TheStreamId, endStream = false, ByteString("0000"))
         entityDataIn.expectUtf8EncodedString("0000")
         pollForWindowUpdates(500.millis)
 
-        sendDATA(TheStreamId, endStream = false, ByteString("1111"))
+        val bigData = ByteString("1" * 100000) // more than configured request-entity-chunk-size, so that something is left in buffer
```

Comment:
This is necessary to avoid the difference between Akka 2.5 and 2.6 which seems to be in buffering (maybe related to `Flow.filter` changes in 2.6).

Reply:
if it's `big data` it must be good :D

Reply:
Indeed!

```suggestion
        val BIGDATAAAAAA = ByteString("1" * 100000) // more than configured request-entity-chunk-size, so that something is left in buffer
```